ixofoundation/ixo-webclient
# Audricus 306 template

## Scope
Correct Template Type shield

## Work Done
https://ixo.atlassian.net/browse/WEB-306

## Steps to Test
check type selection when create template
check type badge after create template

## Screenshots
https://www.loom.com/share/b76de0a0d05a473db025879fb7e8108f
